ferveo-tdec/benches/tpke.rs

```diff
@@ -5,7 +5,7 @@ use ark_ec::pairing::Pairing;
 use criterion::{
     black_box, criterion_group, criterion_main, BenchmarkId, Criterion,
 };
-use group_threshold_cryptography_pre_release::{
+use ferveo_tdec::{
```

Comment:
This remind me of a situation a couple of months ago that was making me crazy when working with Ferveo until I realized (or actually, you pointed out). Even though the package and directory are called `ferveo-tdec`, the crate is automatically renamed to `ferveo_tdec`. Do you think it would be worth to rename everything to underscores?

Reply:
Using both `-` and `_` as a separator in crate names is fine. There is a [debate](https://github.com/rust-lang/api-guidelines/discussions/29) on setting the naming convention.

I'm fine with renaming all crates, we have to republish them anyway. The directories should remain named according to the `kebab-case`.

Reply:
I've documented this as a todo here: https://github.com/nucypher/ferveo/issues/122

Reply:
> The directories should remain named according to the kebab-case

Why?

Reply:
Well, I can't cite any specific source on that, but I think it's a [`cargo` project convention](https://doc.rust-lang.org/cargo/guide/project-layout.html). I think, very generally speaking, top-level modules (including `*.rs` files) are to use `kebab-case` and non-top-level files (never directories) are to use `snake_case`.